_data/support-packages.yaml

```diff
@@ -6,7 +6,7 @@
   image: basic-support.svg
 - package_name: Basic <br>Support
   description: >
-    This is the default support level for new cloud.gov agreements. This package includes all services provided in Platform Support, plus 10hrs of support time that our engineers can devote to resolving issues with your specific applications.
+    This is the default support level for new cloud.gov agreements. This package includes all services provided in Platform Support, plus 10hrs of support time that our engineers can devote to resolving issues with your specific applications.<br /><br />
```

Comment:
why line breaks here? I don't want to insert HTML into YAML fields. If we need vertical space, CSS should do that for us.

Reply:
It's a carry over from the cg-site here:  https://github.com/cloud-gov/cg-site/blob/main/_data/support-packages.yml

I'll update to a CSS solution.